src/components/LoginDialog.jsx

```diff
@@ -10,14 +10,18 @@ import PropTypes from 'prop-types';
 
 const uiConfig = {
   signInFlow: 'popup',
+  signInSuccessWithAuthResult() {
+    return true;
+  },
+  signInSuccessUrl: '/get-started',
```

Comment:
This file's changes do not seem related to this PR

Reply:
what should I do? remove them or keep them

Reply:
Remove them from this PR as they're not related.